Plugins/VvQueryService/KipQueryService/CMakeLists.txt

```diff
@@ -43,6 +43,9 @@ vg_add_qt_plugin(${PROJECT_NAME}
   ${vvKipQueryService_UiSources}
 )
 
+set_property(TARGET ${PROJECT_NAME}
+  PROPERTY CXX_STANDARD 17)
```

Comment:
Why is this needed?

Reply:
Because of the use of `[[maybe_unused]]` here:
https://github.com/neal-siekierski/vivia/blob/viame/dev/windows_fixes/Plugins/VvQueryService/KipQueryService/vvKipQuerySession.cxx#L462
Without it, VS complains that `/std:c++17` is required.

Reply:
Sigh...

Removing the `[[maybe_unused]]` and adding a `Q_UNUSED(pluginsLoaded)` might be better. AFAIK this code is notionally supposed to work on GCC 4.8 which only "mostly" supports C++11 (although also some C++14 bits).